Tests/scripts/configure_tests.py

```diff
@@ -347,17 +375,100 @@ def collect_changed_ids(integration_ids, playbook_names, script_names, modified_
         playbook_names.add(new_playbook)
 
     affected_ids_string = ""
-    if script_names:
-        affected_ids_string += 'Scripts:\n' + '\n'.join(script_names) + '\n\n'
-    if playbook_names:
-        affected_ids_string += 'Playbooks:\n' + '\n'.join(playbook_names) + '\n\n'
-    if integration_ids:
-        affected_ids_string += 'Integrations:\n' + '\n'.join(integration_ids) + '\n\n'
+    if script_names or deprecated_scripts_message:
+        affected_ids_string += 'Scripts:\n' + '\n'.join(script_names) + '\n' + deprecated_scripts_message + '\n'
+    if playbook_names or deprecated_playbooks_message:
+        affected_ids_string += 'Playbooks:\n' + '\n'.join(playbook_names) + '\n' + deprecated_playbooks_message + '\n'
+    if integration_ids or deprecated_integrations_message:
+        affected_ids_string += 'Integrations:\n' + '\n'.join(integration_ids) + '\n' + \
+                               deprecated_integrations_message + '\n'
 
     print('The following ids are affected due to the changes you made:\n{}'.format(affected_ids_string))
+
+    if deprecated_commands_message:
+        print(deprecated_commands_message)
+
     return tests_set, catched_scripts, catched_playbooks
 
 
+def exclude_deprecated_scripts(script_set, script_names):
+    """Removes deprecated scripts from the affected scripts list.
+
+    :param script_set: The set of existing scripts within Content repo.
+    :param script_names: The names of the affected scripts in your change set.
+
+    :return: deprecated_message - A message specifying of all the deprecated scripts.
+    """
+    deprecated_message = ''
+    deprecated_scripts_string = ''
+
+    for script in script_set:
+        script_name = list(script.values())[0].get('name', '')
+        if script_name in script_names:
+            script_data = list(script.values())[0]
+            if script_data.get('deprecated', False):
+                deprecated_scripts_string += '   ' + script_name + '\n'
+                script_names.remove(script_name)
+
+    if deprecated_scripts_string:
+        deprecated_message = '  The following scripts are deprecated and are not taken ' \
+                             'into account in the test collection:\n{}\n'.format(deprecated_scripts_string)
+
+    return deprecated_message
```

Comment:
i think that all of those functions are generic, maybe we should create one function for all of them